app/app.go

```diff
@@ -501,6 +501,7 @@ func NewApp(
 		evmutilSubspace,
 		app.bankKeeper,
 		app.accountKeeper,
+		authtypes.NewModuleAddress(communitytypes.ModuleAccountName).String(),
```

Comment:
does this mean the evm module will only be able to modify funds from the community pool? Can this only ever be one address / is it possible to modify this authorization as part of a governance proposal?